Makefile

```diff
@@ -0,0 +1,53 @@
+####### Universal ######
+
+pip:
+	pip install -e .
+
+pip-test: pip
+	pip install -e.[test]
+
+
+unit-test:
+	pytest -sv tests/unit
+
+blackbox-smoke-prefix:
+	echo "hello blackbox!"
+	pwd
+	ls -l
+	ls -l sandbox
+	cd sandbox && docker-compose ps
+
+integration-test:
+	pytest -sv tests/integration
+
+
```

Comment:
@tzaffi For my understanding:  How is graviton published for downstream consumption (e.g. pyteal)?

I suspect the mechanism exists here, but I'm unfamiliar with the tooling.  I raise the question because none of the `make` targets seem to correlate.

Reply:
It's confusing. But basically as long as:
* you use `pip` to install from this python project (whether from PyPi for Github)
* you have the correct project structure (in particular the `./graviton/` directory has to have all the `__init__.py` files
* you have the correct `setup.py` (using `find_modules()`) 
then everything works automagically.

But that's exactly why I'm not particularly enamored. 

I can see the point of **convention over configuration** if the convention wasn't so clunky...

Reply:
@tzaffi Thanks for the follow up here.  Resolving.